crates/compiler/builtins/bitcode/src/str.zig

```diff
@@ -811,13 +864,40 @@ pub fn strSplit(string: RocStr, delimiter: RocStr) callconv(.C) RocList {
     return list;
 }
 
+const Init = fn (slice_bytes: [*]u8, len: usize, ref_ptr: usize) RocStr;
```

Comment:
this const is unused

Reply:
Oh, yeah, I guess I never use the type. Just functions of the type. removed.